src/components/AdventuresVol/adventures-vol.style.js

```diff
@@ -13,7 +13,19 @@ export const AdventuresVolWrapper = styled.div`
 	  overflow: hidden;
 	  height: 19rem;
 	  position: relative;
-		transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
+	  transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
+
+	  @media (max-width: 450px) {
```

Comment:
@krishkumar84 please avoid using so much of these media queries and also at every width points. these are hard to maintain. Try to use max width and width together. Even if its needed to use it should be only one and used in such a way it should effect the whole component.